### DISCUSSION TITLE
What new things for 1.5?

### DISCUSSION THREAD

**Discussion by paperclipity** [paperclipity/AtlanticHurricaneSimulator#3] [Polls]
I don't get paid to write something here.

**Comment by Defualted** [comment]
I think a better UI would help, maybe show the month while in-game or make a progressable year throughout the game. And you could make a button after you die to show the whole track of the time alive.

**Reply by Defualted** [reply]
And maybe after you become a depression, instead of dying. How about the depression moves around for a bit and if it hits warm water it can upgrade back into a controllable depression.

**Reply by Defualted** [reply]
And maybe buff the landfall affection to where the hurricane loses more mph faster.
